apps/urls.py

```diff
@@ -58,7 +58,8 @@
     ),
     url(r'^admin/', admin.site.urls),
     url(f'{API_PREFIX[1:]}/documents/events', documents.S3Events.as_view(), name='document-events'),
-    url(f'{API_PREFIX[1:]}/devices/status/', shipments.ShipmentOverviewListView.as_view(), name='devices-status'),
```

Comment:
Sorry, I wasn't clear. I meant that we should deprecate this endpoint in favor of the new one. Not remove it entirely. It's important to maintain API compatibility until our clients are all 100% migrated to the new endpoint.